waffle/tests/test_views.py

```diff
@@ -23,14 +26,17 @@ def test_waffle_status(self):
         self.assertIn('max-age=0', cache_control)
 
     def test_waffle_status_response(self):
-        get_waffle_flag_model().objects.create(name='test_flag_active', everyone=True)
```

Comment:
It doesn't seem anything actually needed to be changed here. Revert changes to this section.

Reply:
Again, just revert everything in this section since it's not necessary. Line 33 is the source of your test failure.

Reply:
Ahhhh, I see it now. Sorry about that.